apps/client/components/ui/PrimaryButton.tsx

```diff
@@ -4,13 +4,24 @@ interface Props {
   readonly children: string
   readonly className?: string
   readonly href: string
+  readonly isFormTypeButton?: boolean
 }
 
-export const PrimaryButton = ({ children, className, href }: Props) => {
-  return (
+export const PrimaryButton = ({ children, className, href, isFormTypeButton }: Props) => {
+  return isFormTypeButton ? (
```

Comment:
Tu też powinny to być dwa osobne komponenty, jeden LinkButton (nazwa dowolna), który renderuje Linka, a drugi PrimaryButton który reenderuje button.